free-programming-books-pl.md

```diff
@@ -154,3 +180,8 @@
 
 * [PL/SQL - podstawy (na stronie)](http://andrzejklusiewicz.blogspot.com/2010/11/kurs-oracle-plsql.html)
 * [SQL - podstawy (na stronie)](http://andrzejklusiewicz.blogspot.com/2010/11/kurs-oracle-sql.html)
+
+
+### Inne
+* [Online Regex Tester](https://regex101.com) - Świetna stronka do testowania / budowania regexów.
```

Comment:
This does not appear to be in Polish.

It would be appropriate for the free-programming-playgrounds list, I think.